tests/compare_ncfile.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import os
+import sys 
+import re
+import numpy as np
+##from __future__ import print_function
+from netCDF4 import Dataset
+
+nc1 = Dataset(sys.argv[1])
+nc2 = Dataset(sys.argv[2])
+for varname in nc1.variables.keys():
+    data1 = nc1[varname][:]
```

Comment:
what is the timing on these?  I think you can speed it up further if you don't copy into local variables.  Especially for large arrays.